Changelog.md

```diff
@@ -5,13 +5,14 @@ Features:
  * Code Generator: More specialized and thus optimized implementation for ``x.push(...)``
  * Commandline interface: Error when missing or inaccessible file detected. Suppress it with the ``--ignore-missing`` flag.
  * General: Support accessing dynamic return data in post-byzantium EVMs.
+ * Inheritance: Error when using empty parenthesis for base class constructors that require arguments as experimental 0.5.0 feature.
```

Comment:
If one moves this entry in this pull request one might as well move the one about the ``constructor`` keyword below as well. And change the emberrassing *parenthesis* typo along the way ;-).

Reply:
I will undo the move. We can sort when we release.